db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
```

Comment:
I am not sure if this should be included, but it was its own commit, just in case.